src/styles.css

```diff
@@ -1,398 +1,218 @@
-* {
-    margin: 0px;
-    padding: 0px;
-}
-.row_title{
-  border-left: 1px solid rgb(253, 253, 253);
-  padding: 0.5rem;
-   
-}
-
-
-*{
-  font-family: "Mukta", sans-serif;
-  font-weight: 400;
-  font-style: normal;
-}
-
-
-
-body::-webkit-scrollbar {
-    width: 0.4rem;
-height:40px;
-    color: #000000;
-
-}
-  
-::-webkit-scrollbar-thumb {
-    background-color: rgb(211, 209, 209);
-    border-radius: 10px;
-    height:40px;
-    
-}
-
+/* General Styles */
 body {
-    font-family: 'Netflix Sans', sans-serif;
-    background-color: #111;
-     top: 20px;
-    left: 20px;
-    bottom: 20px;
-    overflow-y: scroll;
-
-}
-
-
-.nav {
-    position: fixed;
-    top: 0;
-    padding: 5px 10px;
-    width: 100%;
-    display: flex;
-    align-items: center;
-    justify-content: space-between;
-    z-index: 1;
-    transition-timing-function: ease-in;
-    transition: all .5s;
-
-}
-
-.nav.active {
-    background-color: rgba(37, 37, 37, .61);
-}
-
-.nav_left {
-    margin: 10px;
-    width: 50%;
-    display: flex;
-}
-
-.nav_logo {
-    width: 7rem;
-
-}
-
-
-.noList li {
-    margin-left: 20px;
-
-    align-items: center;
+    font-family: 'Mukta', sans-serif;
+    margin: 0;
+    background-color: #141414;
```

Comment:
This css code is for the scroll effect of each movie genre row. To scroll in x direction
![image](https://github.com/user-attachments/assets/02418950-559b-4285-b715-e7bdd19ab25c)
```suggestion
    keep the css code asities
```